pkg/endpoint/endpoint.go

```diff
@@ -1360,7 +1363,8 @@ func FilterEPDir(dirFiles []os.FileInfo) []string {
 	eptsID := []string{}
 	for _, file := range dirFiles {
 		if file.IsDir() {
-			if _, err := strconv.ParseUint(file.Name(), 10, 16); err == nil {
+			_, err := strconv.ParseUint(file.Name(), 10, 16)
+			if err == nil || strings.HasSuffix(file.Name(), "_next_fail") {
```

Comment:
This is fine but would a regex be more clear? and capture all the variations more clearly? shrug. I'm not sure if it's worth it.